### PR TITLE
O3de extras ar

### PR DESCRIPTION
**Pull request by byrcolin**

### CODE REVIEW

File: Gems/XR/Code/Tests/XRTest.cpp [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzTest/AzTest.h>
+#include "XRTest.h"
+
+#if !AZ_TRAIT_DISABLE_ALL_XR_TESTS
```
Comment (amzn-phist) [inline Gems/XR/Code/Tests/XRTest.cpp:12]: In OpenXRVkTests.cpp the trait there begins with `O3DE`.  Here we should replace the `AZ` with `O3DE`.

File: Gems/OpenXRVk/Code/Tests/OpenXRVkTests.cpp [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+}
+
+#if O3DE_TRAIT_DISABLE_FAILED_OPENXRVK_TESTS
+TEST_F(OpenXRVkTestFixture, DISABLED_ExpectTrue)
+#else
+TEST_F(OpenXRVkTestFixture, ExpectTrue)
+#endif // O3DE_TRAIT_DISABLE_FAILED_OPENXRVK_TESTS
+{
+    EXPECT_TRUE(false);
```
Comment (amzn-phist) [inline Gems/OpenXRVk/Code/Tests/OpenXRVkTests.cpp:31]: Just testing AR runs here, right?  This would be corrected before checkin?

**Review by amzn-phist — APPROVED** [review]

File: Gems/OpenXRVk/Code/Tests/OpenXRVkTests.cpp (hunk 2 above)
Comment (byrcolin) [reply Gems/OpenXRVk/Code/Tests/OpenXRVkTests.cpp:35]: Yes this is just testing

**Review by moraaar — APPROVED** [review]
LGTM.

Although I wonder if we are making it a bit overcomplicated to start with by PALifying the tests, having traits per platform and having macros to disable failing tests. Is this the default stuff is created by the scripts? Seems like an overkill.

File: Gems/OpenXRVk/Code/Tests/OpenXRVkTests.cpp (hunk 2 above)
Comment (lumberyard-employee-dm) [reply Gems/OpenXRVk/Code/Tests/OpenXRVkTests.cpp:35]: Looks like the place holder were submitted and should not have been.

**Pull request merged**